modules/atomic-widgets/style/style-render.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Elementor\Modules\AtomicWidgets\Style;
+
+class Style_Render {
```

Comment:
```suggestion
class Styles_Renderer {
```

Maybe?
(don't forget the file name)

Reply:
Done